app/models/manageiq/providers/embedded_ansible/automation_manager/job.rb

```diff
@@ -98,13 +98,47 @@ def self.reconcile_extra_vars_keys(_template, options)
   private_class_method :reconcile_extra_vars_keys
 
   def update_with_provider_object(raw_job)
-    self.miq_task ||= raw_job.miq_task
+    transaction do
```

Comment:
For what it was worth, I was trying to keep this as close to the original in `manageiq-providers-ansible_tower`, but that isn't to say this isn't a worth while change.  Just that this makes it a bit of an unknown and less battle tested than the existing alternative.

Your call though, just stating my rational.  :wave:

Reply:
Ah ok...yeah, as I went thorough I realized with the individual updates, it's probably better this became a transaction.